test/integration/associations/has-many.test.js

```diff
@@ -26,6 +26,35 @@ describe(Support.getTestDialectTeaser('HasMany'), () => {
     });
   });
 
+  describe('count', () => {
+    it('the COUNT() attribute should be `{Model.name}.{Model.primaryKeyField}`', function() {
```

Comment:
No stubs, its should be a proper integration test hitting database and asserting on returned data

Reply:
Or use this [year old test](https://github.com/sequelize/sequelize/pull/6115#discussion_r69303807) as suggested by @janmeier